autogpts/forge/forge/config/config.py

```diff
@@ -251,7 +251,9 @@ def assert_config_has_openai_api_key(config: Config) -> None:
         logger.info(
             "You can get your key from https://platform.openai.com/account/api-keys"
         )
-        openai_api_key = clean_input("Please enter your OpenAI API key if you have it:")
+        openai_api_key = click.prompt(
+            "Please enter your OpenAI API key if you have it:"
+        )
```

Comment:
`click.prompt` with no options doesn't behave the same as `clean_input`

Reply:
https://github.com/Significant-Gravitas/AutoGPT/pull/7106/commits/2833815db3a26f7cdafa7cbe9ed197699e84c995